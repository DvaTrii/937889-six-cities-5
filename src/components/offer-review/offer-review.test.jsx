import React from "react";
import renderer from "react-test-renderer";
import OfferReview from "./offer-review";

it(`Should OfferReview render properly`, () => {
  const review = {
    id: 300,
    date: 1605357744252,
    userInfo: {
      avatar: `img/avatar-max.jpg`,
      name: `Tom`
    },
    text: `Lorem ipsum dolor sit amet`,
    rating: 4
  };

  const tree = renderer
    .create(
        <OfferReview
          review={review}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
