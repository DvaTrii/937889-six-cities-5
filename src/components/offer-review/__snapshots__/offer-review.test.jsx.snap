// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OfferReview render properly 1`] = `
<li
  className="reviews__review"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="img/avatar-max.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Tom
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Lorem ipsum dolor sit amet
    </p>
    <time
      className="reviews__time"
      dateTime="Sat Nov 14 2020"
    >
      November
       
      2020
    </time>
  </div>
</li>
`;
